infra/gravity/config.go

```diff
@@ -81,6 +81,8 @@ type ProvisionerConfig struct {
 
 	// ScriptPath is the path to the terraform script or directory for provisioning
 	ScriptPath string `yaml:"script_path" validate:"required"`
+	// TerraformPluginDir
```

Comment:
Good catch.  This useless comment was definitely not intentional... probably something vestigial from my development.

Reply:
> Also, as an idea - if we need to set up terraform from the root configuration, why not add a terraform-specific block and also set the default value when instantiating said root config?

I considered this, but it felt like a bigger change than I wanted to make in this PR... mainly because I'm trying to keep things small in scope as I learn go and the code base.  I've filed #179 to look at doing this in a better way.

Reply:
Addressed in 917b7be.